src/asyncWithLDProvider.tsx

```diff
@@ -7,9 +7,6 @@ import getFlagsProxy from './getFlagsProxy';
 import wrapperOptions from './wrapperOptions';
 import ProviderState from './providerState';
 
-// Extend the expect with additional methods.
```

Comment:
This was the mistake. Intended to go into the test file. Encountered some other dependency problems after moving it, so added those as well.